components/blocks/FilesToCards.vue

```diff
@@ -151,24 +157,40 @@
           width="medium"
         >
           <template #header>
-            <span v-if="item.group" class="radius-0 tag is-link has-text-light"
-              ><abbr :title="item.group | expandAcronym">
-                {{ item.group }}
-              </abbr></span
-            >
+            <span v-for="tagArray in subsetCardTags[i]" :key=tagArray>
+              <span v-for="tag in tagArray" :key=tag class="radius-0 tag m-1" :class="tagColors[cardColors[tag]]">
+                {{ tag }}
+              </span>
+            </span>
             <h2 class="title has-text-black pt-3">{{ item.title }}</h2>
             <div v-if="item.date">Updated: {{ item.date }}</div>
-            <span v-if="item.authors" class="subtitle has-text-black"
-              ><i class="mdi mdi-account-multiple p-1 m-1"></i>
-              <span v-for="(author, index) in item.authors" :key="author.name"
-                ><a
-                  v-if="author.github_user"
-                  :href="'https://github.com/' + author.github_user"
-                  >{{ author.name }}</a
-                >{{ author.github_user ? '' : author.name
-                }}<span v-if="index + 1 < item.authors.length">, </span>
+            <div v-if="item.investigators" >
+              <span class="subtitle has-text-black"
+                ><div><i class="mdi mdi-brain p-1 m-1"></i></div>
+                <span
+                  v-for="(author, index) in item.investigators"
+                  :key="author.name"
+                  ><a v-if="author.link" :href="author.link">{{
+                    author.name
+                  }}</a
+                  >{{ author.link ? '' : author.name
+                  }}<span v-if="index + 1 < item.investigators.length">, </span>
+                </span>
```

Comment:
this is very repetitive with people, make an outer template that loops over `{investigators: 'mdi-brain', people: 'mdi-account-multiple'}` to then generate the html in one go.  There will need to be the special handling for link/github user as well.  You could alternately think about pulling this out to a little component if that would be easier.